spec/factories/card.rb

```diff
@@ -11,4 +12,9 @@
     f.crypted_password "$2a$10$Y7wUSADeOiZd94mIiOB4LOgNEljImLOY/YwQeUy275i6uIVkL9K8i"
     f.salt "B1D8NG63C3SiZjPk1TLv"
   end
+
+   factory :assoc_cat, class: "Category" do |f|
+    f.name "das das das"
+    f.active false
+  end
```

Comment:
`end` at 19, 2 is not aligned with `factory :assoc_cat, class: "Category" do |f|` at 16, 3